Core1Platform/HelpAndSupport/01_MPS.md

```diff
@@ -0,0 +1,14 @@
+# Managed professional services (MPS)
+
+
+With almost 20 years of experience in ERP and e-commerce, Actindo is a strong partner to support you in your growth. You choose from our portfolio how we can support you in a custom-fit way.
+With the Managed Professional Service Packages, you choose from our portfolio how we can support you in a custom-fit way with customized service content such as the following:
```

Comment:
**You choose from our portfolio how we can support you in a custom-fit way.** With the Managed Professional Service Packages, **you choose from our portfolio how we can support you in a custom-fit** way with **customized** service content such as the following:

Ein Satz wiederholt sich; anstatt customized, personalized, um Wiederholung zu vermeiden?